nikola/conf.py.in

```diff
@@ -396,6 +428,19 @@ CATEGORY_OUTPUT_FLAT_HIERARCHY = ${CATEGORY_OUTPUT_FLAT_HIERARCHY}
 # Category pages will still be generated.
 HIDDEN_CATEGORIES = []
 
+# A list of dictionaries specifying categories which translate to each other.
+# For example:
+#   [
+#     {'en': 'private', 'de': 'Privat'},
+#     {'en': 'business', 'fr': 'Arbeit'},
```

Comment:
LOL, yes, it is. I wanted to look up the French word, but apparently I forgot to do that... Fixed that.